K2Field.K2NE.ServiceBroker/ServiceObjects/EmailTemplate/SmoClientHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using SourceCode.SmartObjects.Client;
+
+namespace K2Field.K2NE.ServiceBroker.ServiceObjects.EmailTemplate
+{
+    public class SmoClientHelper : SmartObjectClientServer, IDisposable
```

Comment:
I'm not sure if this class is maybe too much. It's easy to avoid this class altogether and stay more to the K2 API. Will result in mode code, but i'm reasonably ok with that. This class will make it less transparent IMHO.